apps/studio/components/interfaces/SignIn/SignInForm.tsx

```diff
@@ -14,7 +14,9 @@ import { Button, Form, Input } from 'ui'
 
 const signInSchema = object({
   email: string().email('Must be a valid email').required('Email is required'),
-  password: string().required('Password is required'),
+  password: string()
+    .required('Password is required')
+    .max(72, 'Password must be at most 72 characters'),
```

Comment:
@hf Should we be validating here on sign in? Will gotrue choke if it receives a string longer than 72 characters?